petastorm/spark/spark_dataset_converter.py

```diff
@@ -0,0 +1,50 @@
+from petastorm import make_batch_reader
+from petastorm.tf_utils import make_petastorm_dataset
+from pyspark.sql.dataframe import DataFrame
+
+import numpy as np
+import os
+import tensorflow as tf
+
+assert(tf.version.VERSION == '1.15.0')
+
+# Config
+CACHE_DIR = "/tmp/tf"
+
+# primitive type mapping
+type_map = {
+    "boolean": np.bool_,
+    "byte": np.int8,  # 8-bit signed
+    "double": np.float64,
+    "float": np.float32,
+    "integer": np.int32,
+    "long": np.int64,
+    "short": np.int16
+}
+
+
+def save_df_as_parquet(df: DataFrame, dir_path: str) -> str:
+    """
+    Save a SparkDataFrame in parquet format in dbfs (Liang: during dev: the local file system).
+    :param df: SparkDataFrame
+    :param dir_path: the directory for the saved parquet file, could be local, hdfs, dbfs, ...
+    :return: the path of the saved parquet file
+    """
+    df.write.mode("overwrite") \
+      .option("parquet.block.size", 1024 * 1024) \
+      .parquet(CACHE_DIR)
```

Comment:
should be `os.join(CACHE_DIR, dir_path)`
Later we should isolate each python process to use its own root CACHE_DIR. We could update it later.

Reply:
I mentioned it in the PR description:
> 2. It caches all the dataframes to the same fixed path CACHE_DIR, and overwrites the folder every time. Cache management will be added in following PRs.

Reply:
oh. ok.